### PR TITLE
Add "Clean" command

### PR DESCRIPTION
**Pull request by AnaLucyGDL**
Create the command "clean" to delete all the .json files generated into the "assets" directory, and send a message to the user when there are no files to delete.

### CODE REVIEW

File: commands/cfe.js [hunk 1]
```diff
@@ -1,36 +1,41 @@
...
+const SEND_MESSAGE = require('../utils/sendTableMessage');
+const SCRAPPER = require('../webScraping/cfeScrapper');
 
 /**
  * Main function of the command
  * @param { Message } message Channel to send the data
  * @arg { searchItems } args Searching parameters
  * @return { void }
  */
+<<<<<<< HEAD
```
Comment (tris460) [inline commands/cfe.js:14]: Delete it

File: commands/cfe.js [hunk 2]
```diff
@@ -1,36 +1,41 @@
...
+async function execute(message, ARGS) {
+  if (!ARGS || ARGS.length == 0) {
+=======
 async function execute(message, args) {
   if (!args || args.length === 0) {
+>>>>>>> bce62c39820daaabf8d738a20d7c9949cd59ff51
```
Comment (tris460) [inline commands/cfe.js:20]: Delete it

File: commands/clean.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
+module.exports = {
+  name: 'clean',
+  description: 'Erase all JSON files',
+  cooldown: 0,
+  execute(message, ARGS) {
+
+    const fs = require('fs').promises
```
Comment (tris460) [inline commands/clean.js:7]: Add ';'

File: commands/clean.js [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+    const path = require('path')
+    const FOLDER_TO_REMOVE = './assets'
+
+    fs.readdir(FOLDER_TO_REMOVE)
+      .then(files => {
+        console.log('Path')
```
Comment (tris460) [inline commands/clean.js:13]: Do you need this ?

File: commands/clean.js [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+        const unlinkPromises = files.map(file => {
+          const filePath = path.join(FOLDER_TO_REMOVE, file)
+          return fs.unlink(filePath)
+        })
+        message.reply('Files removed succesfully')
+        return Promise.all(unlinkPromises)
+      }).catch(err => {
```
Comment (tris460) [inline commands/clean.js:20]: I think, ESLint/Prettier uses parenthesis in the parameters 
(err)=>{...}

File: scrapperFunctions/index.js [hunk 6]
```diff
@@ -80,9 +80,19 @@ ScrapPage.prototype.clickButton = async function (id, time) {
  * each element a cel
  */
 
-const tableToArrays = (selector) => {
-  const selection = `${selector} tr`;
+const TABLE_TO_ARRAYS = (selector) => {
+  const SELECTION = `${selector} tr`;
   // Get the data with this selector
+<<<<<<< HEAD
```
Comment (tris460) [inline scrapperFunctions/index.js:86]: Delete it

File: scrapperFunctions/index.js [hunk 7]
```diff
@@ -91,13 +101,14 @@ const tableToArrays = (selector) => {
     for (let j = 0; j < elements.length; j++) {
       if (el[j].innerText === '' || el[j].innerText === 'undefined') {
         indArr.push('---');
+>>>>>>> bce62c39820daaabf8d738a20d7c9949cd59ff51
```
Comment (tris460) [inline scrapperFunctions/index.js:104]: Delete it

File: webScraping/cfeScrapper.js [hunk 8]
```diff
@@ -9,23 +9,23 @@
...
-const nextPageBtn = 'div.row a.k-link span.k-i-arrow-e';
-const tableSelector = 'table.k-selectable';
-const rowSelector = '#totProc';
+const PUPPETEER = require('puppeteer');
+const SCRAP_PAGE = require('../scrapperFunctions/index');
+const URL_PAGE = 'https://msc.cfe.mx/Aplicaciones/NCFE/Concursos/';
+const ID_INPUT = '#descProc';
+const ID_BUTTON = '#buscar';
+const WAITING_TIME = 2000;
+// const ROUTE = './assets/Data-From-Table.json';
```
Comment (tris460) [inline webScraping/cfeScrapper.js:18]: I have deleted this line a lot of times, but you always put it again, delete it please

File: webScraping/cfeScrapper.js [hunk 9]
```diff
@@ -79,57 +79,61 @@ Scrapper.prototype.printPercentage = function (percentage) {
...
-      waitingTime,
+    const DATA = await MY_PAGE.checkData(
+      TABLE_SELECTOR,
+      ROW_SELECTOR,
+      NEXT_PAGE_BTN,
+      WAITING_TIME,
       // eslint-disable-next-line prettier/prettier
       this.printPercentage,
     );
+<<<<<<< HEAD
```
Comment (tris460) [inline webScraping/cfeScrapper.js:106]: Delete it

File: commands/cfe.js [hunk 10]
```diff
@@ -1,36 +1,41 @@
 'use strict';
 
-const Validation = require('../utils/validator');
-const myValidator = new Validation();
-const sendMessage = require('../utils/sendTableMessage');
-const Scrapper = require('../webScraping/cfeScrapper');
+const VALIDATION = require('../utils/validator');
```
Comment (tris460) [inline commands/cfe.js:3]: This changes are from the other PR

**Review by tris460 — CHANGES_REQUESTED** [review]
You have a lot of commentaries from the conflicts jaja

File: scrapperFunctions/index.js [hunk 11]
```diff
@@ -80,9 +80,19 @@ ScrapPage.prototype.clickButton = async function (id, time) {
  * each element a cel
  */
 
-const tableToArrays = (selector) => {
-  const selection = `${selector} tr`;
+const TABLE_TO_ARRAYS = (selector) => {
```
Comment (tris460) [inline scrapperFunctions/index.js:83]: You mustn't have this

File: scrapperFunctions/index.js [hunk 12]
```diff
@@ -107,9 +118,9 @@ const tableToArrays = (selector) => {
  * @returns {int | string} The data obtained
  */
 
-const getRows = (selector) => {
-  const rows = document.querySelector(selector).innerText;
-  return rows;
+const GET_ROWS = (selector) => {
```
Comment (tris460) [inline scrapperFunctions/index.js:121]: Delete this function

File: commands/clean.js (hunk 4 above)
Comment (AnaLucyGDL) [reply commands/clean.js:13]: No, was for a test.

**Review by wolftrax5 — APPROVED** [review]

**Pull request merged**